plugins/catalog/src/js/repositories/repositoriesStream.js

```diff
@@ -0,0 +1,32 @@
+import { request } from "@dcos/http-service";
```

Comment:
I like the pattern of moving the request logic to own "libraries", is the `plugins/catalog` folder the place for this kind of thing going forward, or is there something about repositories specifically that dictates this?

Reply:
We have a plugin SDK that loads stuff from plugins. This particular code doesn't implement a plugin interface, but is in plugins directory so that it is separated once we get to modularization and actually start pulling plugins out of the repo.